docs/src/components/LayoutDocsNtmy.js

```diff
@@ -0,0 +1,185 @@
+import * as React from 'react'
+import { MDXProvider } from '@mdx-js/react'
+import NavBar from 'components/NavBar'
+import { Sidebar } from 'components/Sidebar'
+import { SidebarCategory } from 'components/SidebarCategory'
+import { SidebarHeading } from './SidebarHeadingNtmy'
+import { SidebarMobile } from 'components/SidebarMobile'
+import { SidebarPost } from 'components/SidebarPostNtmy'
+import { Sticky } from 'components/Sticky'
+import { useIsMobile } from 'components/useIsMobile'
+import { findRouteByPath } from 'lib/docs/findRouteByPath'
+import { removeFromLast } from 'lib/docs/utils'
+import { getRouteContext } from 'lib/get-route-context'
+import { useRouter } from 'next/router'
+import s from './markdown.ntmy.module.css'
+import FooterDark from './FooterDark'
+import { DocsPageFooter } from './DocsPageFooterNtmy'
+import { Seo } from './Seo'
+import MDXComponents from './MDXComponents'
+import Head from 'next/head'
+import { getManifest } from 'manifests/getManifest'
+import StarButton from './StarButton'
+
+const getSlugAndTag = (path) => {
+  const parts = path.split('/')
+
+  if (parts[2] === '1.5.8' || parts[2] === '2.1.4') {
+    return {
+      tag: parts[2],
+      slug: `/docs/${parts.slice(2).join('/')}`,
+    }
+  }
+
+  return {
+    slug: path,
+  }
+}
+
+const addTagToSlug = (slug, tag) => {
+  return tag ? `/docs/${tag}/${slug.replace('/docs/', '')}` : slug
+}
+
+export const LayoutDocsNtmy = (props) => {
+  const router = useRouter()
+  const { slug, tag } = getSlugAndTag(router.asPath)
+  const { routes } = getManifest(tag)
+  const _route = findRouteByPath(removeFromLast(slug, '#'), routes) // @ts-ignore
+  const { route, prevRoute, nextRoute } = getRouteContext(_route, routes)
+  const title = route && `${route.title}`
+  const isMobile = useIsMobile()
+
+  return (
+    <>
+      {tag && (
+        <Head>
+          <meta name="robots" content="noindex" />
+        </Head>
+      )}
+      <div>
+        {isMobile ? (
+          <>
+            <NavBar />
+            <Sticky shadow>
+              <SidebarMobile>
+                <SidebarRoutes isMobile={true} routes={routes} />
+              </SidebarMobile>
+            </Sticky>
+          </>
+        ) : (
+          <Sticky>
+            <NavBar />
+          </Sticky>
+        )}
+        <Seo
+          title={props.meta.title || title}
+          description={props.meta.description}
+        />
+        <div className="block">
+          <>
+            <div className="container mx-auto pb-12 pt-6 content">
+              <div className="flex relative">
+                {!isMobile && (
+                  <Sidebar fixed>
+                    <SidebarRoutes routes={routes} />
```

Comment:
there is a warning in developer console saying there are duplicate keys
<img width="1141" alt="Screen Shot 2021-06-24 at 16 59 19" src="https://user-images.githubusercontent.com/5974862/123243908-a38ab600-d50d-11eb-83d3-bced3c22f3f9.png">